kolena/dataset/evaluation.py

```diff
@@ -75,6 +77,23 @@ class EvalConfigResults(NamedTuple):
     results: pd.DataFrame
 
 
+@dataclass(frozen=True)
+class ModelEntity:
```

Comment:
Intentionally not using the [internal definition of model entity](https://github.com/kolenaIO/kolena/blob/4c74dd3146ee6ef0816dff12799721aada2d9c21/kolena/_api/v2/model.py#L67) to make this a publicly available class with docs. Mainly to surface the `tags` to users. Skipping `id` here but we can always add it later if needed.